common/azure-common/src/test/java/com/azure/common/implementation/util/FluxUtilTests.java

```diff
@@ -251,4 +265,19 @@ public void toByteArrayWithNonEmptyByteBuffer() {
         return bytes;
     }
 
+    /**
+     * Helper class for creating file if not exists
+     *
+     * @param fileName
+     * @return File file
+     * @throws IOException
+     */
+    private File createFileIfNotExist(String fileName) throws IOException{
+        File file = new File(fileName);
+        if (!file.exists()) {
+            file = Files.createFile(Paths.get(fileName)).toFile();
+        }
+        return file;
```

Comment:
1. We can avoid file existence check here, `File::createNewFile` api can create a file if it does not exists
2. we can remove the test `setUp()` method with `file.getParentFile().mkdirs();` in our `createFileIfNotExist` method
```java
private static File createFileIfNotExist(String fileName) throws IOException {
    File file = new File(fileName);
    if (file.getParentFile() != null) {
        file.getParentFile().mkdirs();
    }
    file.createNewFile();
    return file;
}
```

Reply:
Thanks for suggestions. Reformat as your suggestion.